ui/raidboss/data/06-ew/raid/p1n.txt

```diff
@@ -0,0 +1,191 @@
+### P1N: Asphodelos: The First Circle
+#
+# -ii 65F2 65E6 65DA 65D9 65E0 65DF
+
+# Note: Gaoler's Flail cast is reworded to Aetherflail (65E0|65DF) in the
+# timeline as that is the cast that is visible. However, damage is from the
+# Gaoler's Flail (6DA3|6DA2) cast end after.
+
+# Note: unknown_6d15 happens same time as marker for the stack
+# to avoid knockback mechanic.
+
+hideall "--Reset--"
+hideall "--sync--"
+hideall "unknown_6d15"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+
+## First Gaoler's Flail
+8.7 "--sync--" sync / 14:[^:]*:Erichthonios:6DA3:/ window 10,9 jump 108.7
+8.7 "--sync--" sync / 14:[^:]*:Erichthonios:6DA2:/ window 10,9 jump 208.7
+17.4 "Gaoler's Flail Left/Gaoler's Flail Right?" #sync / 1[56]:[^:]*:Erichthonios:(6DA3|6DA2):/
+28.5 "Gaoler's Flail Right/Gaoler's Flail Left?" #sync / 1[56]:[^:]*:Erichthonios:(6DA2|6DA3):/
```

Comment:
It does simplify and "shortens" the timeline to remove the blocks. I have removed them.

Reply:
When I changed to using 14 from 1[56] in 73a5471, the test timeline would report a lot of missed syncs after jumps. I saw there being an issue in the docs about jumps, so I guess that's what I ran into...

Reply:
Right now, test timeline reports if any syncs are missed.  If you have a sync on 1, 2, 3, 4, 5, and you jump from 1 to 5, it will report that 2, 3, 4 were missed.  We currently don't have any way to differentiate "this is a jump sync, I don't care about missed jumps" and "this is a normal sync, you shouldn't miss anything".